src/types.js

```diff
@@ -19,18 +79,46 @@ export const TYPES = {
     name: 'Community Guidelines',
     fileName: 'community_guidelines'
   },
+  customerPrivacyPolicy: {
+    name: 'Costomer Privacy Policy',
+    fileName: 'costumer_privacy_policy'
+  },
+  DMCA: {
+    name: 'DMCA',
+    fileName: 'dmca'
+  },
+  EULA: {
+    name: 'EULA',
+    fileName: 'eula'
+  },
   brandGuidelines: {
     name: 'Brand Guidelines',
     fileName: 'brand_guidelines'
   },
+  geoGuidelines: {
+    name: 'Geo Guidelines',
+    fileName: 'geo_guidelines'
+  },
   reviewGuidelines: {
     name: 'Review Guidelines',
     fileName: 'review_guidelines'
   },
+  cookiePolicy: {
+    name: 'Cookie Policy',
+    fileName: 'cookie_policy'
+  },
+  cookieAndAdPolicy: {
+    name: 'Cookie and Ad Policy',
+    fileName: 'cookie_and_ad_policy'
+  },
   cookiesPolicy: {
     name: 'Cookies Policy',
     fileName: 'cookies_policy'
   },
+  copyright: {
```

Comment:
Could we have examples, in order to understand if this is about describing copyright owners, about DMCA, or about providing copyright licenses?